src/core/agent/modules/signify/signifyApi.ts

```diff
@@ -103,13 +103,25 @@ export class SignifyApi {
     return { signifyName, identifier: operation.serder.ked.i };
   }
 
-  async interactDelegation(signifyName: string, delegatePrefix: string) {
+  async interactDelegation(
+    signifyName: string,
+    delegatePrefix: string
+  ): Promise<boolean> {
     const anchor = {
       i: delegatePrefix,
       s: "0",
       d: delegatePrefix,
     };
-    return this.signifyClient.identifiers().interact(signifyName, anchor);
+    const ixnResult = await this.signifyClient
+      .identifiers()
+      .interact(signifyName, anchor);
+    const operation = await ixnResult.op();
+    await this.waitAndGetDoneOp(
+      operation,
+      this.opTimeout,
+      this.opRetryInterval
+    );
+    return operation.done;
```

Comment:
Hmm, I'd prefer if `signifyApi` just returns the long running operation I think so the calling function can decide to wait to not. Do we need this change?

Reply:
Actually, nevermind - we don't return the operation for the other ones... I will re-consider it later. The tests should be updated to reflect that this now returns a boolean though